src/krux/pages/home.py

```diff
@@ -87,6 +160,11 @@ def wallet(self):
             self.print_qr_prompt(wallet_data, qr_format)
         return MENU_CONTINUE
 
+    def _binary_seed_qr(self):
+        binary_seed = bip39.mnemonic_to_bytes(self.ctx.wallet.key.mnemonic)
```

Comment:
This isn't the right way to generate a Compact SeedQR. They drop the checksum bits at the end. Check out my implementation here:
https://github.com/selfcustody/krux/blob/3e1a77a6440339c938fb13962c28eadae3df6c78/src/krux/pages/home.py#L62-L68